### PR TITLE
Add OZ v2 relayer

### PR DESCRIPTION
**Pull request by Hyferion**

### CODE REVIEW

**Review by gosuto-inzasheru — APPROVED** [review]
eoa used to (automatically) update rate on layer zero. lgtm

File: extras/signers.json [hunk 1]
```diff
@@ -45,13 +45,15 @@
     "davgarai": "0xAc1aA53108712d7f38093A67d380aD54B562a650"
   },
   "ozDefender": {
-    "checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7"
+    "checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7",
```
Comment (Tritium-VLK) [inline extras/signers.json:48]:
```suggestion
    "checkpointerV1_old": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7",
```

File: extras/signers.json [hunk 2]
```diff
@@ -45,13 +45,15 @@
...
+    "checkpointer_v2": "0xfb7d0D001BC8D0bC998071C762BfF53EE31b725F"
```
Comment (Tritium-VLK) [inline extras/signers.json:49]:
```suggestion
    "relayerV1": "0xfb7d0D001BC8D0bC998071C762BfF53EE31b725F"
```

Comment (Tritium-VLK) [reply extras/signers.json:49]: Checkpointer is the wrong nomenclature here and is an artifact of this being added to our addressbook before we had all the context?  What do you think about doing it something like this to not propagate such foolishness?

File: extras/signers.json [hunk 3]
```diff
@@ -45,13 +45,15 @@
...
   },
   "maxiPay": {
     "factory": "0x7bbac709a9535464690a435ca7361256496f13ce"
   },
   "keepers": {
     "oz_defender_checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7",
+    "oz_v2_defender_checkpointer" : "0xfb7d0D001BC8D0bC998071C762BfF53EE31b725F",
```
Comment (Tritium-VLK) [inline extras/signers.json:56]: I wonder if we need this in 2 places, or if it makes sense to just put all the keepers here?

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]
Can we rename checkpointer to relayer.  That's a more accurate word.  I just used checkpointer the first time because I wasn't really sure what the address was for other than checkpointing/what it was connected to. 

Could also rename the old checkpointer checkpointerOld

File: extras/signers.json (hunk 2 above)
Comment (gosuto-inzasheru) [reply extras/signers.json:49]: there are lots of labels that could be finetuned in this way, but not sure of the (breaking) implications though

File: extras/signers.json (hunk 1 above)
Comment (Hyferion) [reply extras/signers.json:48]: Does this have any implications down the road if we rename the old checkpointer?

Then I would suggest let's call them
old_relayer_v1
relayer_v2

**Review by Tritium-VLK — APPROVED** [review]

**Pull request merged**